training/culture-and-process/70_20_10_TIME.md

```diff
@@ -0,0 +1,44 @@
+70% 20% 10% Time in DP
+===========================
+
+As part of working in Digital Publishing, the development team share sprint time between feature work (70%) driven by product managers,
+technical workflows and tools (20%) driven by Technical Leads and development teams and self training (10%) driven by a developer.
+
+### Pre-reading
+
+- It is useful to understand [Trello Board - Flow of Stories](./TRELLO_BOARD_FLOW.md) and how we use it to organise development workflow as it is mentioned in this documentation
+
+### 70% Time
+
+70% time work includes:
+- Product features
+- Project (epics)
+- High Priority Bugs and Security Maintenance of applications/infrastructure
+
+### 20% Time
+
+20% time is generally used to work on stories that will improve our working experience within digital publishing.
+Any work that needs to happen whether it is a dependency or blocks feature work should be handled in 70% time.
+
+Some things that are covered in 20% time work includes:
+- Support Documentation
+- Streamlining Dev Workflows
+- Improve local dev setup
+- Training Materials
+- Dp Tooling
+
+### 10% Time
+
+10% time can be accumulated across sprints, to be able to spend a few days working to build, research or learn.
+There is the "dev-10-pc-time" channel to post any learnings or progression of prototypes to share with the wider team.
```

Comment:
Probably need to specific that its a Slack channel

Reply:
And also say this isn't mandatory. We wouldn't want not being comfortable posting about what you're learning to stop you from learning it